training/general/src/environments/azureml-lightgbm-3.2-ubuntu20.04-py37-cpu/tests/lightgbm_sample_test.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Tests running a sample job in the lightgbm 3.2 environment."""
+import os
+import time
+from pathlib import Path
+from azure.ai.ml import command, Input, MLClient
+from azure.ai.ml._restclient.models import JobStatus
+from azure.ai.ml.entities import Environment, BuildContext
+from azure.identity import AzureCliCredential
+
+BUILD_CONTEXT = Path("../context")
+JOB_SOURCE_CODE = "src"
+TIMEOUT_MINUTES = os.environ.get("timeout_minutes", 40)
+STD_LOG = Path("artifacts/user_logs/std_log.txt")
+
+
+def test_sklearn_0_24():
```

Comment:
shouldn't this be renamed to mean we're testing LightGBM?

Reply:
yes, should be updated now